app/main/views/stats.py

```diff
@@ -51,10 +51,10 @@ def view_statistics(framework_slug):
             }
         }),
         services_by_lot=format_snapshots(snapshots, 'services', {
-            'IaaS': {'lot': 'IaaS'},
-            'PaaS': {'lot': 'PaaS'},
-            'SaaS': {'lot': 'SaaS'},
-            'SCS':  {'lot': 'SCS'},
+            'iaas': {'lot': 'iaas'},
+            'paas': {'lot': 'paas'},
+            'saas': {'lot': 'saas'},
+            'scs':  {'lot': 'scs'},
```

Comment:
I think you'd need to make the comparison case-insensitive [here](https://github.com/alphagov/digitalmarketplace-admin-frontend/blob/lowercase-lot-updates/app/main/helpers/sum_counts.py#L22) for this to work, because the snapshots will still contain the camelCase lots.

Not sure if we care about the historic G7 data at this point.